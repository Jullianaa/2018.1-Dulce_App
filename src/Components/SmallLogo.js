import React from 'react';
import {View, Image} from 'react-native';
const logo = require('../../assets/img/logo.png');

const styles = {
  logo: {
    alignSelf: 'center',
    marginBottom: 20,
    width: 200,
    height: 100,
    resizeMode: 'contain',
    backgroundColor: 'transparent'
  }
};

class SmallLogo extends React.Component {
  render() {
    return (
      <View>
        <Image source={logo} style={styles.logo} />
      </View>
    );
  }
}

export default SmallLogo;
