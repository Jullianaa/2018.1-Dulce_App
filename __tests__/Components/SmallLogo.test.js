import React from 'react';
import renderer from 'react-test-renderer';
import SmallLogo from '../../src/Components/SmallLogo';
require('bezier');

it('renders correctly', () => {
  const tree = renderer
  .create(<SmallLogo />)
  .toJSON();
  expect(tree).toMatchSnapshot();
});
