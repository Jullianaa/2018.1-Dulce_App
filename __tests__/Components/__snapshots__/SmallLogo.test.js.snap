// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <Image
    source={
      Object {
        "testUri": "../../../assets/img/logo.png",
      }
    }
    style={
      Object {
        "alignSelf": "center",
        "backgroundColor": "transparent",
        "height": 100,
        "marginBottom": 20,
        "resizeMode": "contain",
        "width": 200,
      }
    }
  />
</View>
`;
